galasa-extensions-parent/dev.galasa.auth.couchdb/src/test/java/dev/galasa/auth/couchdb/internal/beans/TestUserDoc.java

```diff
@@ -0,0 +1,278 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package dev.galasa.auth.couchdb.internal.beans;
+
+import static org.assertj.core.api.Assertions.*;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+import org.junit.Test;
+
+import dev.galasa.framework.spi.auth.IFrontEndClient;
+import dev.galasa.framework.spi.auth.IUser;
+
+
+public class TestUserDoc {
```

Comment:
There shouldn't be a need to test a model class since they shouldn't have any logic inside them